wire-ios/WireCommonComponents/FontBook.swift

```diff
@@ -17,9 +17,14 @@
 //
 
 import UIKit
+import SwiftUI
 
 extension UIFont {
 
+    public var swiftUIFont: Font {
+        Font(self as CTFont)
+    }
```

Comment:
While this probably works it's not really moving in direction of using SwiftUI first hand.
Can we use a direct mapping like this?

```swift
    public static func swiftUIFont(for style: UIFont.FontStyle) -> Font {
        switch style {
        case .title3:
            return .title3

        case .headline:
            return .headline

        case .body:
            return .body

        case .subheadline:
            return .subheadline

        case .caption1:
            return .caption

        case .title3Bold:
            return .title3.bold()

        case .calloutBold:
            return .callout.bold()

        case .footnoteSemibold:
            return .footnote.weight(.semibold)

        case .bodyTwoSemibold:
            return .system(size: 16, weight: .semibold)

        case .buttonSmallSemibold:
            return .system(size: 14, weight: .semibold)

        case .buttonBigSemibold:
                return .title3.bold()

        case .bigHeadline:
            return .system(size: 36)
        }
    }
```

Reply:
It would be even better to rename the current func to `uiFont(for:)` and name this one just `font(for:)`

Reply:
> It would be even better to rename the current func to `uiFont(for:)` and name this one just `font(for:)`

Let's keep that as is. That would bring a lot of changed files.